mat64/cblas_test.go

```diff
@@ -0,0 +1,15 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//+build cblas
+
+package mat64
+
```

Comment:
Does this mean the package is tested with cblas? Is this file somehow avoided with the go test call?